public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -228,7 +228,7 @@ export class TempoDatasource extends DataSourceWithBackend<TempoQuery, TempoJson
           });
 
           if (config.featureToggles.traceQLStreaming) {
-            subQueries.push(this.handleStreamingSearch(options, targets.traceql));
+            subQueries.push(this.handleStreamingSearch(options, targets.traceql, queryValue));
```

Comment:
can you elaborate why this fixes this? The change does not make it obvious why sending an optional param changes this.